.prettierrc.json

```diff
@@ -1,6 +1,9 @@
 {
-    "trailingComma": "none",
-    "tabWidth": 4,
+    "tabWidth": 2,
+    "useTabs": false,
     "semi": true,
-    "bracketSpacing": true
+    "singleQuote": true,
+    "trailingComma": "all",
+    "printWidth": 100,
```

Comment:
fra https://prettier.io/docs/en/options.html: "For readability we recommend against using more than 80 characters"

Reply:
Var usikker på denne, så flott at du kommenterer. Skal fikse i neste push :)